packages/app/src/cli/services/dev/theme-extension-args.ts

```diff
@@ -0,0 +1,38 @@
+import {AppInterface} from '../../models/app/app.js'
+import {ensureDeployEnvironment} from '../environment.js'
+
+export async function themeExtensionArgs(
+  apiKey: string,
+  options: {app: AppInterface; theme?: string; port?: number; reset: boolean},
+) {
+  const {identifiers} = await ensureDeployEnvironment(options)
```

Comment:
Problem is, we can't have this during the `dev` flow, it will essentially break it (because of the validations we do in `ensureDeployEnvironment`)
We will need to find a way to obtain those IDs without calling `ensureDeployEnvironment`. I can take a look tomorrow on alternatives and maybe we can pair and discuss the solution :)

Reply:
Awesome, let's do that :)